src/dodal/devices/slits/slit_motor.py

```diff
@@ -0,0 +1,72 @@
+from collections import OrderedDict
+from enum import IntEnum
+
+from bluesky.protocols import Movable, Readable
+from ophyd import Component, Device, EpicsSignal, EpicsSignalRO
+from ophyd.status import SubscriptionStatus
+
+from dodal.devices.status import await_value
+
+
+class MoveState(IntEnum):
+    Moving = (0,)
+    Stationary = 1
+
+
+class SlitMotor(Device, Movable, Readable):
+    """A class representing a slit motor.
+
+    This could be gap, centre, x, y, etc...
+    By default sets to .VAL and reads from .RBV
+
+    Unimplemented signals:
+        SPMG
+        KILL
+        Anything under "More"
+    """
+
+    value: EpicsSignal = Component(EpicsSignal, ".VAL")
+    readback_value: EpicsSignalRO = Component(EpicsSignalRO, ".RBV")
+
+    tweak_forward: EpicsSignal = Component(EpicsSignal, ".TWF")
+    tweak_reverse: EpicsSignal = Component(EpicsSignal, ".TWR")
+    tweak_value: EpicsSignal = Component(EpicsSignal, ".TWV")
+
+    done_moving: EpicsSignalRO = Component(EpicsSignalRO, ".DMOV")
+    alarm_severity: EpicsSignalRO = Component(EpicsSignalRO, ".SEVR")
+
+    stop_motor: EpicsSignal = Component(EpicsSignal, ".STOP")
```

Comment:
Should: See Ophyd's [`EpicsMotor`](https://nsls-ii.github.io/ophyd/builtin-devices.html#ophyd.epics_motor.EpicsMotor). It defines most of this stuff for you. You should probably inherit from that instead of `Device`